volatility3/framework/plugins/windows/info.py

```diff
@@ -3,15 +3,35 @@
 #
 
 import time
+import struct
 from typing import List, Tuple, Iterable
 
 from volatility3.framework import constants, interfaces, layers, symbols
 from volatility3.framework.configuration import requirements
-from volatility3.framework.interfaces import plugins
+from volatility3.framework.interfaces import plugins, configuration
 from volatility3.framework.renderers import TreeGrid
 from volatility3.framework.symbols import intermed
 from volatility3.framework.symbols.windows import extensions
+from volatility3.framework.layers import physical
 
+def rol(value: int, count: int, max_bits: int = 64) -> int: 
+    """A rotate-left instruction in Python"""
+    max_bits_mask = (1 << max_bits) - 1
+    return (value << count % max_bits) & max_bits_mask |  \
+           ((value & max_bits_mask) >> (max_bits - (count % max_bits)))
+
+def bswap_32(value: int):
+    value &= 0xffffffff
+    value = ((value << 8) & 0xFF00FF00) | ((value >> 8) & 0x00FF00FF)
+
+    return ((value << 16) | (value >> 16)) & 0xffffffff
+
+def bswap_64(value: int) -> int:
+    value &= 0xffffffffffffffff
```

Comment:
So we spent time changing `rol` not to use hardcoded limited numbers, but they've snuck back into bswap_32 and bswap_64 in a big way.  I'm also not sure why the doing pack and unpack with a changed endianness didn't achieve the same goal?  Could we go back to the pack/unpack method and use a flag to choose whether the struct should be `I` for 32 bit of `Q` for 64 please?